src/pytorch_lightning/core/module.py

```diff
@@ -1471,8 +1471,12 @@ def clip_gradients(
         """Handles gradient clipping internally.
 
         Note:
-            Do not override this method. If you want to customize gradient clipping, consider
-            using :meth:`configure_gradient_clipping` method.
+            - Do not override this method. If you want to customize gradient clipping, consider using
+            :meth:`configure_gradient_clipping` method.
+            - For manual optimization (``self.automatic_optimization = False``), if you want to use
+            gradient clipping, consider calling
+            ``self.clip_gradients(opt, gradient_clip_val=0.5, gradient_clip_algorithm="norm")``
+            manually in the training step.
```

Comment:
One more small issue: 

```
Just one more small issue to fix: /home/runner/work/lightning/lightning/src/pytorch_lightning/core/module.py:docstring of pytorch_lightning.core.module.LightningModule.clip_gradients:6: WARNING: Bullet list ends without a blank line; unexpected unindent.
```


```suggestion
            - Do not override this method. If you want to customize gradient clipping, consider using
              :meth:`configure_gradient_clipping` method.
            - For manual optimization (``self.automatic_optimization = False``), if you want to use
              gradient clipping, consider calling
              ``self.clip_gradients(opt, gradient_clip_val=0.5, gradient_clip_algorithm="norm")``
              manually in the training step.
```